### PR TITLE
Brock/core #60

### PR DESCRIPTION
**Pull request by blchelle**
## Description
Added the setting tab which currently does the following

* Gets the user info and displays it
* Provides a button to edit user information, when pressed a dialogue will appear (Submitting doesn't change info yet since the authentication controller is needed to change users email)
* Provides a button to log the user out. (Not implemented but looks like it will extremely trivial once I can access the auth controller in the SettingsFragment)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     android:name=".BoromiApp"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
-    android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+    android:theme="@style/AppTheme">
```
Comment (akchen1) [inline app/src/main/AndroidManifest.xml:12]: Why is this changed? Changing this will break our current bar

Comment (blchelle) [reply app/src/main/AndroidManifest.xml:12]: It's not actually changing it, I changed AppTheme to Theme.ApppCompat.Light.NoAction bar. This let's android use our defined colours where appropriate.

For example: In the login page, the underline below each text field was the default android color (that dark green color) because we weren't actually applying our theme.

I changed it in this ticket because it was doing the same to the edit_user_fragment

Comment (akchen1) [reply app/src/main/AndroidManifest.xml:12]: o okok

**Review by akchen1 — APPROVED** [review]
lgtm
